cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -185,6 +185,79 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'align', 'justify');
 	});
 
+	it('Apply Line spacing: 1', function() {
+		cy.get('#tb_editbar_item_linespacing')
+			.click();
+
+		helper.reselectAllText();
+		
+		cy.get('.menu-text').eq(0)
+			.click();
+
+		helper.reselectAllText();
+
+		cy.get('#copy-paste-container p')
+			.should('have.attr', 'style', 'margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%');
```

Comment:
This assertion passes even if the line spacing item is not pushed. The document contains 100% line spacing for the selected text by default. If this feature breaks in the future, the test won't catch the regression, because the assertion would pass anyway.
A good way to check whether a test would catch a regression is that if you disable the part of the test which applies the new paragraph formatting on the text. If the test passes without the applying part, then the test is not correctly written.